### ISSUE TITLE
Bug in Safari with accordion and newer version of JQuery

### ISSUE THREAD

**Issue by Martinomagnifico** [zurb/foundation-sites#11640]
## What should happen?

The accordion menu should open and close flawlessly. 
The menu item that is selected should open with increasing height.

## What happens instead?

The menu item that is selected is shown in whole for a flash of a second.
See example at https://gifyu.com/image/w96p and the codepen as mentioned below.

## Possible Solution

The problem is the version of jQuery. The last version that works correctly is 3.1.1. Any newer version will show the strange behaviour.

## Test Case and/or Steps to Reproduce (for bugs)

Test Case: https://codepen.io/martino/pen/LMjzBd

## Context

...

## Your Environment

- Foundation version(s) used: 6.5.1
- Browser(s) name and version(s): Safari 12
- Device, Operating System and version: Desktop iMac

## Checklist

- [x] I have read and follow the CONTRIBUTING.md document.
- [x] There are no other issues similar to this one.
- [x] The issue title and template are correctly filled.

**Comment by Martinomagnifico** [comment]
Yes, it is from 3.1.1 up. I checked with the JQuery versions in the pen. 
Safari is 12.0.2, 12.1 Tech preview shows the same issue.

**Comment by Martinomagnifico** [comment]
The bug is also there on mobile Safari. Can anyone confirm?

**Comment by jamesonnuss** [comment]
For me this issue is happening on both desktop and mobile. The issue doesn't occur if you set data-multi-expand="true" on the accordion but that isn't a solution, just an observation.

**Comment by Martinomagnifico** [comment]
The issue is still there in Foundation 6.5.3. Would it be jQuery or Foundation ?

**Comment by DanielRuf** [comment]
Probably a jQuery regression. Which is the first version which causes this?

Might be a good idea to report it at the jQuery issue tracker.
